nautobot/dcim/forms.py

```diff
@@ -786,6 +786,24 @@ class ManufacturerFilterForm(NautobotFilterForm):
     platforms = DynamicModelMultipleChoiceField(queryset=Platform.objects.all(), to_field_name="name", required=False)
 
 
+#
+# Platform
+#
+class PlatformBulkEditForm(NautobotBulkEditForm):
+    pk = forms.ModelMultipleChoiceField(queryset=Platform.objects.all(), widget=forms.MultipleHiddenInput())
+    description = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    name = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    manufacturer = forms.ModelChoiceField(
+        queryset=Manufacturer.objects.all(), required=False, widget=forms.Select(attrs={"class": "form-control"})
+    )
+    network_driver = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    napalm_driver = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    napalm_args = forms.JSONField(required=False, widget=forms.Textarea(attrs={"rows": 4, "class": "form-control"}))
+
+    class Meta:
+        nullable_fields = ["description", "name", "manufacturer", "network_driver", "napalm_driver", "napalm_args"]
```

Comment:
The only fields with `null=True` in the model definition are `manufacturer` and `napalm_args`; I suspect trying to null out any of the other ones will result in an error.